bin/oref0-determine-basal.js

```diff
@@ -105,7 +106,7 @@ if (!module.parent) {
             console.error(msg.msg);
             // console.log(JSON.stringify(msg));
             if (!params['missing-meal-ok']) {
-              errors.push(msg);
+              warnings.push(msg);
```

Comment:
FWIW, `--no-missing-meal-ok` means "strict" mode which I would prefer throw an error (since it's being asked for).